### PR TITLE
Add changelog entry for colour presets as discussed in #18514

### PR DESCRIPTION
**Pull request by 733737**
This was a request from developers to update the changelog to reflect added ride colour presets.

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 0.4.3 (in development)
 ------------------------------------------------------------------------
 - Feature: [#17782] The Flying Coaster now has access to boosters and can draw outside loops.
+- Feature: [#18013,18016,18018,18019,18514] Added colour presets to Spiral Slide, Dodgems, Boat Hire, Flying Saucers, and Car Ride.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:4]:
```suggestion
- Improved: [#18013, #18016, #18018, #18019, #18514] Added colour presets to Spiral Slide, Dodgems, Boat Hire, Flying Saucers, and Car Ride.
```

Note that you will have to move it down to be in order, and that you have to rebase or use the GitHub-provided option to resolve conflicts.

**Review by Gymnasiast — CHANGES_REQUESTED** [review]

File: distribution/changelog.txt (hunk 1 above)
Comment (733737) [reply distribution/changelog.txt]: I have updated it

**Comment by tupaschoal** [comment]
@733737 can you please solve the merge conflict for this to be merged?

**Comment by 733737** [comment]
done

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**